reccmp/isledecomp/formats/image.py

```diff
@@ -0,0 +1,12 @@
+import dataclasses
+from pathlib import Path
+
+
+@dataclasses.dataclass
+class Image:
+    filepath: Path
+    view: memoryview = dataclasses.field(repr=False)
+    data: bytes = dataclasses.field(repr=False)
```

Comment:
Do we need `data` if we have the memoryview of the file?

Reply:
I added it for extracting the PDB filename from a PE image: `memoryview` has no `index` or `find` method.

Reply:
Ah you're right. Maybe we could just send this as the param to `CodeViewHeaderNB10` and `CodeViewHeaderRSDS`:
```python
debug_raw = bytes(self.view[debug_entry.pointer_to_raw_data : debug_entry.pointer_to_raw_data + debug_entry.size_of_data])
```

Could even use `rstrip` to capture the string if the pdb filename is all that's left.

Reply:
That's possible, but then you'd do a memory copy of potentially a lot of data (but most of the times not much).